File: tuts/004-cloudmap-custom-attributes/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from servicediscovery_wrapper import CloudMapAttrsWrapper


def run_scenario():
    client = boto3.client("servicediscovery")
    wrapper = CloudMapAttrsWrapper(client)
    print(f"Running CloudMapAttrs getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()

File: tuts/010-cloudmap-service-discovery/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 'servicediscovery_wrapper'

def run_scenario
  client = Aws::CloudMap::Client.new
  wrapper = CloudMapWrapper.new(client)
  puts "Running CloudMap getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME
